lib/mongoose_push/service/apns.ex

```diff
@@ -103,36 +90,18 @@ defmodule MongoosePush.Service.APNS do
     config
   end
 
-  defp maybe_setup_default_apns_topic(config) do
-    try do
-      # There are loooots of things that may went wrong with this. Notably, dev certificates
-      # don't have topic list, while production certificates may not have it if they are old enough.
-      # Also the whole `extract_topics!` function is based on reverse-engineered ASN.1 struct,
-      # so there may be some incompability issues that we may work on based on failure logs.
-      case config[:default_topic] do
-        nil ->
-          all_topics = Certificate.extract_topics!(config[:cert])
-          default_topic = all_topics[:topic]
-          Logger.info(~s"Successfully extracted default APNS topic: #{default_topic}")
-          Keyword.put(config, :default_topic, default_topic)
-
-        default_topic ->
-          Logger.info(~s"Using user-defined default APNS topic: #{default_topic}")
-          config
-      end
-    catch
-      _, reason ->
-        Logger.warn(
-          ~s"Unable to extract APNS topic from the #{config[:mode]} certificate " <>
-            "due to: #{inspect(reason)}"
-        )
-
-        config
-    end
-  end
-
   defp maybe_mutable_content(notification, true),
```

Comment:
Maybe this `maybe` should be used instead. The only difference is the argument, but we can make a `maybe` version without argument to pass.